thin-replica-server/include/thin-replica-server/subscription_buffer.hpp

```diff
@@ -175,7 +175,7 @@ class SubUpdateBuffer {
 
   void waitForEventGroupUntilNonEmpty() {
     std::unique_lock<std::mutex> lock(eg_mutex_);
-    eg_cv_.wait(lock, [this] { return queue_.read_available(); });
+    eg_cv_.wait(lock, [this] { return eg_queue_.read_available(); });
```

Comment:
I'm glad we are having tests now. This is the 2nd typo in this file within a short period of time 🐞